mm2src/mm2_main/src/lp_swap/maker_swap_v2.rs

```diff
@@ -2101,6 +2128,7 @@ impl<MakerCoin: MmCoin + MakerCoinSwapOpsV2, TakerCoin: MmCoin + TakerCoinSwapOp
         self: Box<Self>,
         state_machine: &mut Self::StateMachine,
     ) -> <Self::StateMachine as StateMachineTrait>::Result {
+        // FIXME: Do we want to broadcast the swap status here?
```

Comment:
Why not?

Reply:
this covers other cases like (e.g.) a maker who didn't see the funding transaction for some reason and aborted the swap, or a taker that failed to send the funding transaction and aborted the swap. idk if we want statistics about failures like these.

though a swap finished via a refund is accounted for and sent for stats.

Reply:
Reasons for failure could be good intel for future works to mitigate.
Correct, seednodes do collect, in `stats_swaps` table column `is_success`. 
https://dev.komodo-docs.pages.dev/en/docs/komodo-defi-framework/tutorials/query-the-mm2-database/#stats-swaps